orion/evaluation/common.py

```diff
@@ -52,5 +70,7 @@ def _f1_score(expected, observed, data, start, end, cm):
         return 2 * (precision * recall) / (precision + recall)
 
     except ZeroDivisionError:
-        print('Invalid value encountered for precision {}/ recall {}.'.format(precision, recall))
+        LOGGER.exception(
```

Comment:
what if precision or recall = np.nan

Reply:
It would just be `np.nan`; here are some examples
```python3
# precision as nan
>>> tp = np.nan
>>> fp = 1
>>> tp / (tp + fp)
nan

# f1 score will be nan
>>> 2 * (np.nan * 1) / (np.nan + 1)
nan
```

Reply:
So if the output is nan, the outside function will handle nan as well? If that is true, then I think ZeroDivisionError is enough here.

Reply:
Not sure what you mean by "handle" but it won't invoke an error because `anything * np.nan` or `anything + np.nan` will just be a `np.nan` without raising any errors. Then in the benchmark, I will record the value as is `np.nan` because I cannot compute this value.